README.md

```diff
@@ -1 +0,0 @@
-# testing
```

Comment:
It's fine to remove this text, but there needs to be some sort of README. Usually best if it has instructions for building and running whatever code exists